packages/cerebral-provider-firebase/src/createOnValue.js

```diff
@@ -4,16 +4,32 @@ import {
 } from './helpers'
 
 export default function createOnValue (controller) {
-  return (path, signal, options) => {
+  return (path, signal, options = {}) => {
+    let hasEmittedInitialValue = false
     listenTo(
       createRef(path, options),
       path,
       'value',
       signal,
       (data) => {
-        controller.getSignal(signal)({
+        if (!hasEmittedInitialValue) {
+          hasEmittedInitialValue = true
+          return
+        }
```

Comment:
AH, hehe... yeah, seems a bit weird maybe. But the thing is that it should skip the first value :)  Cause you do not care about first value, you care about changes to the value. To grab current value you use `value` :)